reana-ui/src/util.js

```diff
@@ -173,3 +174,12 @@ export function stringifyQueryParams(params) {
     skipEmptyString: true,
   });
 }
+
+/**
+ * Open Jupyter Notebook in a new tab.
+ */
+export const openJupyterNotebook = (event, sessionUri, reanaToken) => {
+  const uri = `${api}${sessionUri}?token=${reanaToken}`;
```

Comment:
If we do not append the token it opens up a login screen where token needs to be entered. We also append it in CLI. This is something to be fixed quite urgently I think

Reply:
I can see what you mean now. It's not cool to expose the token like this when we're already logged in. Maybe we can create a ticket for this?

Reply:
I agree, this is part of a different issue since it touches more broader topics. 

Short explanation: This is due to login living inside REANA-Server (more exactly, inside Invenio). In order to achieve the described behaviour, we would need to:
1. Place an authentication service in front of all of these components (R-Server and interactive sessions) (going more in the direction of having Kubernetes based authentications system).
2. Or have an authentication mechanism that allows federated authentication